spec/feature/unauthenticated_guest_spec.rb

```diff
@@ -121,11 +119,10 @@
                 last_name: "Shea",
                 email: "bryce@gmail.com",
                 password: "secret")
-    visit root_path
+    click_link "Log In"
     fill_in "session[email]", with: "bryce@gmail.com"
     fill_in "session[password]", with: "secret"
-    click_link_or_button "Log In"
-    expect(current_path).to eq(root_path)
+    click_button "Log In"
     within("#cart-contents") do
```

Comment:
Def need to come through and fix the rest of this test with the cart-contents... to be continued.  :sparkles: